enterprise/coderd/groups.go

```diff
@@ -233,15 +242,15 @@ func (api *API) patchGroup(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	members, err := api.Database.GetGroupMembers(ctx, group.ID)
-	if err != nil {
+	patchedMembers, patchedMembersErr := api.Database.GetGroupMembers(ctx, group.ID)
+	if patchedMembersErr != nil {
```

Comment:
Do these errors need to be named differently? It seems like the wrong error is being passed below.

Reply:
Oops! Will fix. What convention do you like for naming errors? Just call them `err` until there's a scoping conflict?

Reply:
Unless you need multiple errors in scope at once, I would always just name them `err`!

Reply:
Sounds good! Will update now.